README.md

```diff
@@ -317,7 +317,14 @@ Here is a minimal Docker image that can run Panther:
 ```
 FROM php:latest
 
-RUN apt-get update && apt-get install -y libzip-dev zlib1g-dev chromium && docker-php-ext-install zip
+# Install PHP zip extension, a dependency for symfony/panther library
+RUN docker-php-ext-install zip
+
+# Install chrome, required for the symfony/panther library
+RUN wget http://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_80.0.3987.122-1_amd64.deb -O /tmp/google-chrome-stable_80.0.3987.122-1_amd64.deb
```

Comment:
As per my comment in #304, this URL can be changed to one that retrieves the latest release. `https://dl.google.com/dl/linux/direct/google-chrome-stable_current_amd64.deb`. 

A subsequent call to `symfony/panther/chromedriver-bin/update.sh` (executed from the directory where it resides) will ensure that chrome-driver matches the installed chrome version.

Reply:
@webignition I have updated README file as requested I agree that download always the stable and running the update.sh is the right approach. In  this PR I am proposing a slight change to the sh file since basically it relies on running CD command before executing it. Otherwise the downloaded files end up in whatever directory you executed the script from. With my changed the script understand where it is and ensure that the new files end up in the same directory.